dockers/docker-database/multi_database_config.json.j2

```diff
@@ -0,0 +1,182 @@
+{% set include_remote_db = (REMOTE_DB_IP is defined and REMOTE_DB_PORT is defined) %}
+{
+    "INSTANCES": {
+        "redis":{
+            "hostname" : "{{HOST_IP}}",
+            "port" : {{REDIS_PORT}},
+            "unix_socket_path" : "/var/run/redis{{DEV}}/redis.sock",
+            "persistence_for_warm_boot" : "yes"
+            {% if DATABASE_TYPE is defined and DATABASE_TYPE != "" %}
+            ,"database_type": "{{DATABASE_TYPE}}"
+            {% endif %}
+        },
+        "redis1": {
+           "hostname" : "{{HOST_IP}}",
+            "port" : 6380,
+            "unix_socket_path" : "/var/run/redis{{DEV}}/redis1.sock",
+            "persistence_for_warm_boot" : "yes"
+            {% if DATABASE_TYPE is defined and DATABASE_TYPE != "" %}
+            ,"database_type": "{{DATABASE_TYPE}}"
+            {% endif %}
+        },
+        "redis2": {
+            "hostname" : "{{HOST_IP}}",
+            "port" : 6381,
+            "unix_socket_path" : "/var/run/redis{{DEV}}/redis2.sock",
+            "persistence_for_warm_boot" : "yes"
+            {% if DATABASE_TYPE is defined and DATABASE_TYPE != "" %}
+            ,"database_type": "{{DATABASE_TYPE}}"
+            {% endif %}
+        },
+        "redis3": {
+           "hostname" : "{{HOST_IP}}",
+            "port" : 6382,
+            "unix_socket_path" : "/var/run/redis{{DEV}}/redis3.sock",
+            "persistence_for_warm_boot" : "yes"
+            {% if DATABASE_TYPE is defined and DATABASE_TYPE != "" %}
+            ,"database_type": "{{DATABASE_TYPE}}"
+            {% endif %}
+        },
+        "redis4": {
+            "hostname" : "{{HOST_IP}}",
+            "bind": "240.1.1.2 127.0.0.1",
```

Comment:
@Pan-XT Why bind to 240.1.1.2?

Reply:
240.1.1.2 is the communication address between the Host and BMC. If the community version does not have BMC, the redis4/BMC_DB can be removed.